src/mindlessgen/qm/xtb.py

```diff
@@ -82,12 +87,22 @@ def optimize(
             if np.any(np.isin(molecule.ati, get_lanthanides())):
                 # Reset the UHF value to the original value before returning the optimized molecule.
                 optimized_molecule.uhf = uhf_original
+            if super_heavy_elements:
+                # Reset the atomic numbers to the original values before returning the optimized molecule.
+                optimized_molecule.ati = ati_original
+                molecule.ati = ati_original
```

Comment:
Why do adjust two different `Molecule` objects at this point?

Reply:
It is a relic from the debugging